src/mantarray_waveform_analysis/transforms.py

```diff
@@ -217,7 +218,38 @@ def calculate_displacement_from_voltage(
     Returns:
         A 2D array of time vs Displacement
     """
-    voltage = voltage_data[1, :]
+    sample_in_millivolts = voltage_data[1, :] * MILLI_TO_BASE_CONVERSION
     time = voltage_data[0, :]
-    displacement = voltage * -1 + 0.1  # simplistically just invert the data for now
-    return np.vstack((time, displacement)).astype(np.float32)
+
+    # calculate magnetic flux density
+    millivolts_per_millitesla = 1073.6
+    sample_in_milliteslas = sample_in_millivolts / millivolts_per_millitesla
+
+    # calculate displacement
+    millimeters_per_millitesla = 23.25
+    sample_in_millimeters = sample_in_milliteslas * millimeters_per_millitesla
+    sample_in_meters = sample_in_millimeters / MILLI_TO_BASE_CONVERSION
+
+    return np.vstack((time, sample_in_meters)).astype(np.float32)
+
+
+def calculate_force_from_displacement(
+    displacement_data: NDArray[(2, Any), np.float32],
+) -> NDArray[(2, Any), np.float32]:
+    """Convert displacement to force.
+
+    Args:
+        displacement_data: time and Displacement numpy array. Typically coming from calculate_displacement_from_voltage
+
+    Returns:
+        A 2D array of time vs Force
+    """
+    sample_in_millimeters = displacement_data[1, :] * MILLI_TO_BASE_CONVERSION
+    time = displacement_data[0, :]
+
+    # calculate force
+    millinewtons_per_millimeter = 0.000159
```

Comment:
typically a constant is declared outside of a function (in the main scope of the module) so that line of code isn't executed every single time the function is called.  normally it's not much of a concern, but we do anticipate that the conversion of voltage data to force, peak detection, and calculating metrics will become a bottleneck area in the performance of the Live View of the software that we'll need to optimize. So just trying to keep tiny best practices like this in mind ahead of time